Makefile

```diff
@@ -53,12 +59,34 @@ clean: ## Removes local containers, images, volumes, etc
 	@docker-compose rm -f -v -s
 	@docker volume rm -f pims-frontend-node-cache
 
-local-client-tests: ## Runs the client tests in a container
+pause-30:
+	@echo "$(P) Pausing 30 seconds..."
+	@-sleep 30
+
+client-test: ## Runs the client tests in a container
 	@echo "$(P) Running client unit tests..."
 	@docker-compose run frontend npm test
 
-local-server-tests: ## Runs the server tests in a container
+server-test: ## Runs the server tests in a container
 	@echo "$(P) Running server unit tests..."
 	@docker-compose run backend dotnet test
 
-.PHONY: local restart run stop build clean local-client-tests local-server-tests
+server-run: ## Starts local server containers
+	@echo "$(P) Starting server containers..."
+	@docker-compose up -d keycloak backend
+
+database-run:
+	@echo "$(P) Starting database container..."
+	@docker-compose up -d database
+
+database-clean: | database-run pause-30 ## Re-creates an empty docker database - ready for seeding
+	@echo "$(P) Refreshing the database..."
+	@cd backend/dal; dotnet ef database drop --force; dotnet ef database update
+
+database-seed: | server-run pause-30
+	@echo "$(P) Seeding docker database..."
+	@cd backend/tools/import; make run
+
+database-refresh: | database-clean database-seed ## Refreshes the docker database
```

Comment:
The idea here is to be able to run `make database-refresh` to automate all the manual steps to refresh your local DB container. I have tried a couple times with good results. Again, I'm using `make` due to seeing other teams at the Lab leverage it.